cmd/webhook/main.go

```diff
@@ -90,26 +90,26 @@ func main() {
 		Options: options,
 		Handlers: map[schema.GroupVersionKind]webhook.GenericCRD{
 			// For group eventing.knative.dev,
-			eventingv1alpha1.SchemeGroupVersion.WithKind("Channel"):            &eventingv1alpha1.Channel{},
-			eventingv1alpha1.SchemeGroupVersion.WithKind("ClusterProvisioner"): &eventingv1alpha1.ClusterProvisioner{},
-			eventingv1alpha1.SchemeGroupVersion.WithKind("Source"):             &eventingv1alpha1.Source{},
-			eventingv1alpha1.SchemeGroupVersion.WithKind("Subscription"):       &eventingv1alpha1.Subscription{},
+			eventingv1alpha1.SchemeGroupVersion.WithKind(system.KindChannel):            &eventingv1alpha1.Channel{},
+			eventingv1alpha1.SchemeGroupVersion.WithKind(system.KindClusterProvisioner): &eventingv1alpha1.ClusterProvisioner{},
+			eventingv1alpha1.SchemeGroupVersion.WithKind(system.KindSource):             &eventingv1alpha1.Source{},
+			eventingv1alpha1.SchemeGroupVersion.WithKind(system.KindSubscription):       &eventingv1alpha1.Subscription{},
 
 			// For group channels.knative.dev,
-			channelsv1alpha1.SchemeGroupVersion.WithKind("Bus"):          &channelsv1alpha1.Bus{},
-			channelsv1alpha1.SchemeGroupVersion.WithKind("ClusterBus"):   &channelsv1alpha1.ClusterBus{},
-			channelsv1alpha1.SchemeGroupVersion.WithKind("Channel"):      &channelsv1alpha1.Channel{},
-			channelsv1alpha1.SchemeGroupVersion.WithKind("Subscription"): &channelsv1alpha1.Subscription{},
+			channelsv1alpha1.SchemeGroupVersion.WithKind(system.KindBus):          &channelsv1alpha1.Bus{},
+			channelsv1alpha1.SchemeGroupVersion.WithKind(system.KindClusterBus):   &channelsv1alpha1.ClusterBus{},
+			channelsv1alpha1.SchemeGroupVersion.WithKind(system.KindChannel):      &channelsv1alpha1.Channel{},
```

Comment:
While the value happens to be the same, it feels wrong to use the same constant for CRDs in different groups.